metrics.go

```diff
@@ -20,27 +20,29 @@ var (
 )
 
 const (
-	ns               = "circuit"
+	subSystem        = "circuit"
 	circuitNameLabel = "name"
 	funcTypeLabel    = "func"
 
 	runCmd      = "run"
 	fallbackCmd = "fallback"
 )
 
-func init() {
+func newMetrics(nameSpace string) {
```

Comment:
1. Could we name this function `initMetrics`? It makes more sense to me.
2. the name of arg should be `namespace`